File: app/src/main/java/com/eagskunst/emmanuel/gamingnews/models/Response.java
package com.eagskunst.emmanuel.gamingnews.models;

import com.google.gson.annotations.SerializedName;

public class Response{

	@SerializedName("date")
	private int date;

	@SerializedName("game")
	private Game game;

	@SerializedName("updated_at")
	private int updatedAt;

	@SerializedName("created_at")
	private int createdAt;

	@SerializedName("y")
	private int Y;

	@SerializedName("id")
	private int id;

	@SerializedName("category")
	private int category;

	@SerializedName("human")
	private String human;

	@SerializedName("m")
	private int M;

	@SerializedName("platform")
	private int platform;

	public void setDate(int date){
		this.date = date;
	}

	public int getDate(){
		return date;
	}

	public void setGame(Game game){
		this.game = game;
	}

	public Game getGame(){
		return game;
	}

	public void setUpdatedAt(int updatedAt){
		this.updatedAt = updatedAt;
	}

	public int getUpdatedAt(){
		return updatedAt;
	}

	public void setCreatedAt(int createdAt){
		this.createdAt = createdAt;
	}

	public int getCreatedAt(){
		return createdAt;
	}

	public void setY(int Y){
		this.Y = Y;
	}

	public int getY(){
		return Y;
	}

	public void setId(int id){
		this.id = id;
	}

	public int getId(){
		return id;
	}

	public void setCategory(int category){
		this.category = category;
	}

	public int getCategory(){
		return category;
	}

	public void setHuman(String human){
		this.human = human;
	}

	public String getHuman(){
		return human;
	}

	public void setM(int M){
		this.M = M;
	}

	public int getM(){
		return M;
	}

	public void setPlatform(int platform){
		this.platform = platform;
	}

	public int getPlatform(){
		return platform;
	}

	@Override
 	public String toString(){
		return 
			"Response{" + 
			"date = '" + date + '\'' + 
			",game = '" + game + '\'' + 
			",updated_at = '" + updatedAt + '\'' + 
			",created_at = '" + createdAt + '\'' + 
			",y = '" + Y + '\'' + 
			",id = '" + id + '\'' + 
			",category = '" + category + '\'' + 
			",human = '" + human + '\'' + 
			",m = '" + M + '\'' + 
			",platform = '" + platform + '\'' + 
			"}";
		}
}

File: app/src/main/java/com/eagskunst/emmanuel/gamingnews/models/Response.kt
package com.eagskunst.emmanuel.gamingnews.models

import com.google.gson.annotations.SerializedName

class Response {

    @SerializedName("date")
    var date: Int = 0

    @SerializedName("game")
    var game: Game? = null

    @SerializedName("updated_at")
    var updatedAt: Int = 0

    @SerializedName("created_at")
    var createdAt: Int = 0

    @SerializedName("y")
    var y: Int = 0

    @SerializedName("id")
    var id: Int = 0

    @SerializedName("category")
    var category: Int = 0

    @SerializedName("human")
    var human: String? = null

    @SerializedName("m")
    var m: Int = 0

    @SerializedName("platform")
    var platform: Int = 0

    override fun toString(): String {
        return "Response{" +
                "date = '" + date + '\'' +
                ",game = '" + game + '\'' +
                ",updated_at = '" + updatedAt + '\'' +
                ",created_at = '" + createdAt + '\'' +
                ",y = '" + y + '\'' +
                ",id = '" + id + '\'' +
                ",category = '" + category + '\'' +
                ",human = '" + human + '\'' +
                ",m = '" + m + '\'' +
                ",platform = '" + platform + '\'' +
                "}"
    }
}
